### PR TITLE
fix: all remaining fixes for accepting ApePay [SBK-194]

### PR DESCRIPTION
**Pull request by antazoey**

### CODE REVIEW

File: apepay/__init__.py [hunk 1]
```diff
@@ -337,6 +339,7 @@ def funding_rate(self) -> Decimal:
         """
         return Decimal(self.amount_per_second) / Decimal(10 ** self.token.decimals())
 
+    @property
     def funding_estimate(self, period: timedelta) -> int:
```
Comment (fubuloubu) [inline apepay/__init__.py:343]: I added the arg to help you figure out how much the amount per second would be in days or weeks or whatever period you were interested in

Comment (antazoey) [reply apepay/__init__.py:343]: oh I see that now, ok will put that back.
But I am going to keep this PR around to contain all the small fixes I need

Comment (antazoey) [reply apepay/__init__.py:343]: i think it may help to have more method-like name here

Comment (antazoey) [reply apepay/__init__.py]: changed it to `estimate_funding`, so the verb is first

Comment (fubuloubu) [reply apepay/__init__.py]: Nice, just rename the PR

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**